README.md

```diff
@@ -42,6 +42,8 @@ cluster (the Kind tools is configured in the path).
 
 Optional inputs:
 
+- `verbosity`: numeric log verbosity, (info = 0, debug = 3, trace = 2147483647) (default `"0"`)
+- `quiet`: silence all stderr output (default `"false"`)
```

Comment:
Would you please mind adding these two flags at the end of the list?

Reply:
Ok, ! Done